pkg/client/informers/externalversions/factory.go

```diff
@@ -30,8 +33,17 @@ type sharedInformerFactory struct {
 
 // NewSharedInformerFactory constructs a new instance of sharedInformerFactory
 func NewSharedInformerFactory(client versioned.Interface, defaultResync time.Duration) SharedInformerFactory {
+	return NewFilteredSharedInformerFactory(client, defaultResync, v1.NamespaceAll, nil)
+}
+
+// NewFilteredSharedInformerFactory constructs a new instance of sharedInformerFactory.
+// Listers obtained via this SharedInformerFactory will be subject to the same filters
+// as specified here.
+func NewFilteredSharedInformerFactory(client versioned.Interface, defaultResync time.Duration, namespace string, tweakListOptions internalinterfaces.TweakListOptionsFunc) SharedInformerFactory {
 	return &sharedInformerFactory{
 		client:           client,
+		namespace:        namespace,
+		tweakListOptions: tweakListOptions,
```

Comment:
why "tweak" prefix? the ListOption is only used for the informer watch mechanism. 
If it was me, I will have chosen "wathcListOptions", but it is a personal taste :)

Reply:
this is generated code... I didn't choose :)

Reply:
ok, It is strange that they don't put  `_generated.go`as suffix